CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 - Bug #59: Fix rendering nullable union types (@vjik)
 - Bug #62: Fix rendering intersection types (@vjik)
+- Bug #63: Finalize `ObjectProxy::__construct()` (@vjik)
```

Comment:
Bug or Chg?

Reply:
> Bug or Chg?

I think `ObjectProxy` created inside package only, so bug. But `chg` can be also...